mods/cnc/audio/music.yaml

```diff
@@ -108,3 +68,58 @@ recn226m: Recon
 	VolumeModifier: 0.8
 voic226m: Voice Rhythm
 	VolumeModifier: 0.8
+heavyg: Demolition (Heavy Gear)
+	VolumeModifier: 0.7
+
+# SFX
+aoi2: Act On Instinct (SFX)
+	Filename: aoi
+	Extension: var
+befeared2: To Be Feared (SFX)
+	Filename: befeared
+	Extension: var
+deception: We Will Stop Them (SFX)
+	Filename: stopthem
+	Extension: var
+	VolumeModifier: 0.7
+justdoit2: Just Do It Up (SFX)
+	Filename: justdoit
+	Extension: var
+	VolumeModifier: 0.7
+nomercy2: No Mercy (SFX)
+	Filename: nomercy
+	Extension: var
+	VolumeModifier: 0.7
+trouble2: In Trouble (SFX)
+	Filename: trouble
+	Extension: var
+
+# Hidden tracks
+intro: Intro
+	Hidden: true
+	VolumeModifier: 0.9
+map: Map
+	Hidden: true
+	VolumeModifier: 0.6
+map1: Map Theme
+	Hidden: true
+	VolumeModifier: 0.7
+nod_map1: Nod Map Theme
+	Hidden: true
+win1: Great Shot!
+	Hidden: true
+	VolumeModifier: 0.7
+win2: Great Shot! (SFX)
+	Filename: win1
+	Extension: var
+	Hidden: true
+nod_win1: Nod Win Theme
+	Hidden: true
+	VolumeModifier: 0.9
+score: Militant Force (Scores)
```

Comment:
This doesn't belong here. MF was a RA1 track.

Reply:
I will remove this.